### PR TITLE
fix some cli command description

### PR DESCRIPTION
**Pull request by guangxuli**
**what this PR does:**

Mainly fix some description of some `oc` command, especially for the default value of option is false.

when i run `oc deploy -h`, the output : 
```
Options:
      --cancel=false: Cancel the in-progress deployment.
      --enable-triggers=false: Enables all image triggers for the deployment config.
      --follow=false: Follow the logs of a deployment
      --latest=false: Start a new deployment now.
      --retry=false: Retry the latest failed deployment.   
```

I think the annotation of the options is confusing, since all the description are about `* = true` condition,
while the default value of these options are false.  I thought this is our rules at beginning, but when I run `oc new-build -h` command, I saw some outputs are more reasonable, like this(not all):

```
Options:
      --allow-missing-images=false: If true, indicates that referenced Docker images that cannot be found locally or in
a registry should still be used.
      --allow-missing-imagestream-tags=false: If true, indicates that image stream tags that don't exist should still be
used.
      --dry-run=false: If true, show the result of the operation without performing it.
      --no-output=false: If true, the build output will not be pushed anywhere.
```
  I think we should add` if true` when the bool value of description before colons is **false**, that seems more clear.

**what I have fixed:**
1. option=false, but the description is about in case of the value is true, add `if true` to the description if not exist:
```
cmd.Flags().BoolVar(&o.Confirmed, "confirm", o.Confirmed, "**Specify** that cluster role bindings should be modified. Defaults to false, displaying what would be replaced but not actually replacing anything.")
cmd.Flags().BoolVar(&o.Confirmed, "confirm", o.Confirmed, "**If true, specify** that cluster role bindings should be modified. Defaults to false, displaying what would be replaced but not actually replacing anything.")
```
2. description `if present` change it to` if true` just for keeping consistent:
```
cmd.Flags().Bool("all-namespaces", false, "**If present,** list the requested object(s) across all namespaces. Namespace in current context is ignored even if specified with --namespace.")
cmd.Flags().Bool("all-namespaces", false, "**If true**, list the requested object(s) across all namespaces. Namespace in current context is ignored even if specified with --namespace.")
```

**what I do not fixed:**
1. description include all false/true, just like:
```
cmd.Flags().BoolVar(&options.Readiness, "readiness", options.Readiness, "Set or remove a readiness probe to indicate when this container should receive traffic")
cmd.Flags().BoolVar(&options.Liveness, "liveness", options.Liveness, "Set or remove a liveness probe to verify this container is running")
```
2. description have include clear option content, just like:
```
cmd.Flags().BoolVar(&opts.aliasTag, "alias", false, "Should the destination tag be updated whenever the source tag changes. **Defaults to false**.")cmd.Flags().BoolVar(&opts.referenceTag, "reference", false, "Should the destination tag continue to pull from the source namespace. **Defaults to false**.")
```
3.the option have two format, just like:
`-t, --tty=false: Force a pseudo-terminal to be allocated`

### CODE REVIEW

**Comment by pweil-** [comment]
@openshift/cli-review

**Comment by fabianofranz** [comment]
[test]

**Comment by guangxuli** [comment]
@fabianofranz ptal,thanks.

**Comment by guangxuli** [comment]
@fabianofranz I check the results of test failure, it seems a flake. Can you trigger test again? thanks.

**Comment by juanvallejo** [comment]
check flaked on https://github.com/openshift/origin/issues/12157 re[test]

**Comment by guangxuli** [comment]
@juanvallejo thanks.:)

**Comment by guangxuli** [comment]
@juanvallejo thanks.:)

**Comment by guangxuli** [comment]
@juanvallejo thanks.:)

**Comment by guangxuli** [comment]
@juanvallejo test failure again, seems another flake.

**Comment by juanvallejo** [comment]
integration check flaked on https://github.com/openshift/origin/issues/8571
re[test]


@guangxuli looks like you'll need to rebase as well

**Comment by guangxuli** [comment]
@juanvallejo thanks so much, have rebased.

**Comment by juanvallejo** [comment]
@fabianofranz lgtm

re[test]

**Comment by fabianofranz** [comment]
flaked on https://github.com/openshift/origin/issues/8502 re[test]

**Comment by fabianofranz** [comment]
flaked on https://github.com/openshift/origin/issues/8502 re[test]

**Comment by guangxuli** [comment]
@fabianofranz @juanvallejo seems a flake again

**Comment by juanvallejo** [comment]
integration check flaked on https://github.com/openshift/origin/issues/11775 re[test]

File: pkg/cmd/cli/cmd/set/volume.go [hunk 1]
```diff
@@ -171,15 +171,15 @@ func NewCmdVolume(fullName string, f *clientcmd.Factory, out, errOut io.Writer)
 		},
 	}
 	cmd.Flags().StringVarP(&opts.Selector, "selector", "l", "", "Selector (label query) to filter on")
-	cmd.Flags().BoolVar(&opts.All, "all", false, "select all resources in the namespace of the specified resource types")
+	cmd.Flags().BoolVar(&opts.All, "all", false, "If true, Select all resources in the namespace of the specified resource types")
```
Comment (fabianofranz) [inline pkg/cmd/cli/cmd/set/volume.go]: *select must be lowercase.

File: pkg/cmd/admin/router/router.go [hunk 2]
```diff
@@ -270,7 +270,7 @@ func NewCmdRouter(f *clientcmd.Factory, parentName, name string, out, errout io.
 	cmd.Flags().IntVar(&cfg.StatsPort, "stats-port", cfg.StatsPort, "If the underlying router implementation can provide statistics this is a hint to expose it on this port. Specify 0 if you want to turn off exposing the statistics.")
 	cmd.Flags().StringVar(&cfg.StatsPassword, "stats-password", cfg.StatsPassword, "If the underlying router implementation can provide statistics this is the requested password for auth.  If not set a password will be generated.")
 	cmd.Flags().StringVar(&cfg.StatsUsername, "stats-user", cfg.StatsUsername, "If the underlying router implementation can provide statistics this is the requested username for auth.")
-	cmd.Flags().BoolVar(&cfg.ExposeMetrics, "expose-metrics", cfg.ExposeMetrics, "This is a hint to run an extra container in the pod to expose metrics - the image will either be set depending on the router implementation or provided with --metrics-image.")
+	cmd.Flags().BoolVar(&cfg.ExposeMetrics, "expose-metrics", cfg.ExposeMetrics, "If true, this is a hint to run an extra container in the pod to expose metrics - the image will either be set depending on the router implementation or provided with --metrics-image.")
```
Comment (fabianofranz) [inline pkg/cmd/admin/router/router.go]: "f true, attempts to run ..."

**Comment by fabianofranz** [comment]
A couple minor nits then LGTM.

File: pkg/cmd/admin/router/router.go (hunk 2 above)
Comment (guangxuli) [inline pkg/cmd/admin/router/router.go]: agreed. thanks.

File: pkg/cmd/cli/cmd/set/volume.go (hunk 1 above)
Comment (guangxuli) [inline pkg/cmd/cli/cmd/set/volume.go]: will be fixed, thanks.

**Comment by guangxuli** [comment]
@fabianofranz done. PTAL, thanks.

**Comment by fabianofranz** [comment]
Needs a rebase.

**Comment by guangxuli** [comment]
@fabianofranz have rebased.

**Comment by guangxuli** [comment]
@fabianofranz PTAL,thanks.

**Comment by fabianofranz** [comment]
Thanks @guangxuli! LGTM [merge]

**Comment by guangxuli** [comment]
@fabianofranz test failure again, need another trigger.Thanks.

**Pull request merged**